src/components/Loading/Loading.js

```diff
@@ -3,6 +3,23 @@ import PropTypes from 'prop-types';
 
 import { prefixedId } from '../../utils';
 
+export const InlineLoading = ({ text, id }) => (
+  <div id={id} key={prefixedId(id, 'progress') || 'progress'}>
+    <div className="spinner spinner-lg blank-slate-pf-icon" />
+    {text && <h3 className="blank-slate-pf-main-action">{text}</h3>}
```

Comment:
we probably do not want h3 text inside the InlineLoading. I do not think the text is necessary at all.

also, not sure if it would make sense to have a span instead of a div.

Reply:
or 2nd option: having the text inline

Reply:
text is now removed and I also made spinner size configurable